docs/en/Journal.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: global
+title: Journal
+nickname: Journal
+group: Features
+priority: 0
+---
+
+* Table of Contents
+{:toc}
+
+Alluxio maintains a journal to support persistence of metadata operations. 
+When a request modifies Alluxio state, e.g. creating or renaming files, the
+Alluxio master will write a journal entry for the operation before returning
+a successful response to the client. The journal entry is written to a 
+persistent storage such as disk or HDFS, so even if the Alluxio 
+master process is killed, state will be recovered on restart.
+
+# Configuration
+
+The most important configuration value to set for the journal is
+`alluxio.master.journal.folder`. This must be set to a shared filesystem available
+to all masters. In single-master mode, a local filesystem path is fine. With
+multiple masters distributed across different machines, the shared folder should
+be in a distributed system that supports flush such as HDFS or NFS. It is not
+recommended to put the journal in an object store. With an object store, every 
+update to the journal requires a new object to be created, which is 
+prohibitively slow for most serious use cases.
+
+**Configuration examples:**
+```
+alluxio.master.journal.folder=hdfs://[namenodeserver]:[namenodeport]/dir/alluxio_journal
+```
+```
+alluxio.master.journal.folder=/opt/alluxio/journal
+```
+
+# Formatting
+
+Before starting Alluxio for the first time, the journal must be formatted.
+
+**WARNING: formatting the journal will delete all Alluxio metadata**
+```bash
+$ bin/alluxio formatMaster
+```
+
+# Backup
+
+Alluxio supports taking journal backups so that Alluxio metadata can be restored 
+to a previous point in time. Generating a backup causes temporary service
+unavailability while the backup is happening.
+
+To generate a backup, use the `fsadmin backup` CLI command.
+```bash
+$ bin/alluxio fsadmin exportJournal
+```
+
+By default, this will write a backup named 
+`alluxio-journal-YYYY-MM-DD-timestamp.gz` to the "/alluxio_backups" directory of 
```

Comment:
Related question to before, but where is "/alluxio_backups"? in UFS? relative to root, or our mount point? Will it show up in Alluxio namespace eventually?

Reply:
added example and updated docs for the `alluxio.master.backup.directory`